celery_app.py

```
@@ -1,6 +1,8 @@
 from celery import Celery
 
 from config import settings
+from entities.ai_character import AICharacter  # noqa: F401
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)